cluster/manifests/coredns/configmap.yaml

```diff
@@ -15,6 +15,11 @@ data:
             upstream
             fallthrough in-addr.arpa ip6.arpa
         }
+        template IN A skipper.cluster.local  {
+            match "^.*[.]skipper[.]cluster[.]local"
```

Comment:
Should we find a better name than Skipper here? I would like to prevent "leaking" the implementation detail that Skipper is our Ingress proxy to users. I think it would be better to find a meaningful abstract DNS name e.g. `ingress.cluster.local` (or whatever naming idea you would have).

Reply:
You could also consider the „leak“ good, because then they find the right documentation. :)

Reply:
Having Skipper in the DNS name would expose this to all application configurations (which are using service-to-service in the cluster), please don't do this --- it's not about "branding" or documentation here, but about having meaningful endpoints across potentially hundreds of applications.